### DISCUSSION TITLE
Ask for the "PrivacyInfo.xcprivacy" required by Apple

### DISCUSSION THREAD

**Discussion by baihao** [nlohmann/json#4402] [Q&A]
Hi team,

Is there a plan to include the PrivacyInfo.xcprivacy manifest as required by Apple? Please refer to the Apple's document https://developer.apple.com/documentation/bundleresources/privacy_manifest_files for the detailed information, thanks.

Regards,
Hao

**Comment by gregmarr** [comment]
I don't see that this library meets any of the criteria for including it.

**Comment by nlohmann** [comment]
Maybe relevant: https://github.com/abseil/abseil-cpp/commit/4676ffa9811c10d11fc2c782d9399a3f41d739b8 and https://github.com/abseil/abseil-cpp/issues/1602
